pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterActions/DBClusterActions.test.tsx

```diff
@@ -37,25 +37,43 @@ describe('DBClusterActions::', () => {
 
     expect(root.find('button').prop('disabled')).toBeFalsy();
   });
-  it('disables button if cluster changing', () => {
+
+  it('calls delete action correctly', async () => {
```

Comment:
IMO we can still have a test to check if other actions are disabled when the cluster is changing. For example the restart.